File: unit 4_javascript/forloops.js
// For Loop = is a type of loop that iterates through
// a list.
// A list has a finite loop; it has a start point
// and an end point; these are typcially the first
// and last item in the list.

let groceryList = ["apple", "mango", "water", "cereal", "beef"];
let i = 0;
for (i; i < groceryList.length; i += 1) {
  console.log("this is a  " + groceryList[i]);
}

let gradeBookList = [80, 70, 74, 68, 93, 92, 100, 70];
let x = 0;
for (x; x < gradeBookList.length; x += 1) {
  if (gradeBookList[x] < 75) {
    gradeBookList[x] + 5;
    console.log(gradeBookList[x])
  }
  //console.log(gradeBookList[x]);
}

File: unit 4_python/forLoops.py
# FOR LOOP is a type of loop that iterates over
# a list. 

# A for loop does not repeat 
# itself infinitely (or atleast it shouldnt)
# it will loop through the length of the list.

# numbers = [1,2,3,4,5,6,7,8,9,10]
# for x in numbers:
#     print(x)
#     print('loop is repeating...')
#     if x ==10:
#         print('Loop has stopped.')


# groceryList = ['apple','orange','beef','bread','orange','water']

# for item in groceryList:
#     if item == 'orange':
#         continue
#     print(item)


gradeList = [100,50,70,80,80,95,70,50]

for grade in gradeList:
    if grade < 75:
        grade += 5
        print(grade)